develop/parachains/maintenance/unlock-parachain.md

```diff
@@ -0,0 +1,131 @@
+
+2. Query the parachain locked status:
+    1. Select **`registrar`**
+    2. Choose the **`paras`** option
+    3. Input the parachain ID you want to check as a parameter (e.g. `2006`)
+    4. Click the **+** button to execute the query
+    5. Check the status of the parachain lock
+
+    ![](/images/develop/parachains/maintenance/unlock-parachain/unlock-parachain-1.webp)
+
+## How to Unlock a Parachain
+
+Unlocking a parachain requires sending an XCM (Cross-Consensus Message) to the relay chain from the parachain itself, sending a message with Root origin, or this can be accomplished through the relay chain's governance mechanism executing a root call.
+
+If sending an XCM, the parachain origin must have proper authorization, which typically comes from either the parachain's sudo pallet (if enabled) or through its governance system.
+
+This guide demonstrates the unlocking process using a parachain with the sudo pallet. For parachains using governance-based authorization instead, the process will require adjustments to how the XCM is sent.
+
+### Prepare the Unlock Call
+
+Before sending the XCM, you need to construct the relay chain call that will be executed. Follow these steps to prepare the `registrar.removeLock` extrinsic:
+
+1. In [Polkadot.js Apps](https://polkadot.js.org/apps/#/explorer){target=\_blank}, connect to the relay chain, navigate to the **Developer** dropdown and select the **Extrinsics** option
+
+2. Build the `registrar.removeLock` extrinsic
+    1. Select the **registrar** pallet
+    2. Choose the **removeLock** extrinsic
+    3. Fill in the parachain ID parameter (e.g. `2006`)
+    4. Copy the encoded call data
+
+    ![](/images/develop/parachains/maintenance/unlock-parachain/unlock-parachain-2.webp)
+
+3. Determine the transaction weight required for executing the call. You can estimate this by executing the `transactionPaymentCallApi.queryCallInfo` runtime call with the encoded call data previously obtained:
+
+    ![](/images/develop/parachains/deployment/coretime-renewal/coretime-renewal-3.webp)
+
+    This weight information is crucial for properly configuring your XCM message's execution parameters in the next steps.
+
+### Fund the Sovereign Account
+
+For a successful XCM execution, the [sovereign account](https://github.com/polkadot-fellows/xcm-format/blob/10726875bd3016c5e528c85ed6e82415e4b847d7/README.md?plain=1#L50){target=\_blank} of your parachain on the relay chain must have sufficient funds to cover transaction fees. The sovereign account is a deterministic address derived from your parachain ID.
+
+You can identify your parachain's sovereign account using either of these methods:
+
+- Use the **"Para ID" to Address** section in [Substrate Utilities](https://www.shawntabrizi.com/substrate-js-utilities/){target=\_blank} with the **Sibling** option selected
+
+- Calculate it manually:
```

Comment:
there's also a runtime API for that:
https://github.com/ordian/unlock-parachain/blob/721ce67a30824a934c2d5fbb1ea4fda04017d7b1/src/main.rs#L79-L93